swingset/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -62,7 +62,7 @@ org.eclipse.jdt.core.compiler.problem.missingJavadocTags=warning
 org.eclipse.jdt.core.compiler.problem.missingJavadocTagsMethodTypeParameters=disabled
 org.eclipse.jdt.core.compiler.problem.missingJavadocTagsOverriding=disabled
 org.eclipse.jdt.core.compiler.problem.missingJavadocTagsVisibility=private
-org.eclipse.jdt.core.compiler.problem.missingOverrideAnnotation=ignore
+org.eclipse.jdt.core.compiler.problem.missingOverrideAnnotation=warning
```

Comment:
My IDE doesn't care, maybe there's an option to warn. Generating javadoc shows deprecated. So this warning seems pretty useless. It doesn't bother me if its there. I hope this was automated